apps/editing-toolkit/editing-toolkit-plugin/premium-content/editor.css

```diff
@@ -123,6 +130,15 @@
 	margin: 0 8px 0 0;
 }
 
-.editor-styles-wrapper .wp-block-buttons .wp-block[data-type='jetpack/recurring-payments'] .wp-block-button:not( .alignleft ):not( .alignright ) {
+.editor-styles-wrapper
+	.wp-block-buttons
+	.wp-block[data-type='jetpack/recurring-payments']
+	.wp-block-button:not( .alignleft ):not( .alignright ) {
 	margin: 0;
 }
+
+/* Hide Stripe nudge from Jetpack payments block */
```

Comment:
Prettier got excited about formatting this file, but this is the relevant change for this PR.